src/current/v23.2/live-migration-service.md

```diff
@@ -470,34 +484,77 @@ To install `molt-lms-cli`, download the binary that matches your system. To down
+| `-l`, `--lms-addresses` | LMS instances to run the command against. This can be a comma-separated list of IP addresses (e.g., `127.0.0.1`), IP addresses and ports (e.g., `127.0.0.1:1024`), or hostnames (e.g., `https://lms.net`). |
+| `-o`, `--output-type`   | Specify whether `molt-lms-cli` output is formatted in `json` or `table` format.<br><br>**Default:** `table`                                                                                                |
+
+#### `cutover consistent begin` flags
+
+|           Flag          |                                                                                                                                                                                                                      Description                                                                                                                                                                                                                      |
+|-------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--abort-begin-timeout` | Maximum duration to wait before the LMS aborts cutover following a `ctrl-c` command.<br><br>**Default:** `"2s"`                                                                                                                                                                                                                                                                                                                                       |
+| `--begin-timeout`       | Maximum duration to wait before traffic to the LMS is paused for consistent cutover. This should be the approximate length of the longest-running transaction, e.g., `30s`. If no `--begin-timeout` value is specified, the LMS waits for all transactions and queries to finish before pausing traffic.                                                                                                                                              |
+| `-l`, `--lms-addresses` | LMS instances to run the command against. This can be a comma-separated list of IP addresses (e.g., `127.0.0.1`), IP addresses and ports (e.g., `127.0.0.1:1024`), or hostnames (e.g., `https://lms.net`). By default, cutover is performed on all LMS instances deployed in the same environment as the orchestrator. LMS addresses should only be manually specified if the orchestrator is deployed in a different environment than the instances. |
```

Comment:
```suggestion
| `-l`, `--lms-addresses` | LMS instances to run the command against. This can be a comma-separated list of IP addresses (e.g., `127.0.0.1`), IP addresses and ports (e.g., `127.0.0.1:1024`), or hostnames (e.g., `https://lms.net`). By default, cutover is performed on all LMS instances deployed in the same environment as the orchestrator. For example, if all the orchestrator and LMS instances are deployed in the same k8s cluster, the addresses of all LMS instances are automatically detected by the orchestrator and no user configuration is needed. LMS addresses should only be manually specified if the orchestrator is deployed in a different environment than the instances. |
```
@ryanluu12345 might want to confirm if this is accurate too.

Reply:
I think we would want to clarify that its in the same Namespace. https://github.com/cockroachlabs/crdb-proxy/blob/main/orchestrator/api/lms_connections_retriever.go#L45 since that is how we run the query.